TestbedScripts/TestEnvUtils.ps1

```diff
@@ -249,15 +322,14 @@ function Initialize-SimpleEnvironment {
         Write-Host "Yes."
     }
 
-    Run-Agent
+    Initialize-ForwardingRules -MacPhysical $MacPhysical -MacHNSTransparent $MacHNSTransparent -Mac1 $Mac1 -Mac2 $Mac2 -Container1IfName $Container1IfName -Container2IfName $Container2IfName
 }
 
 # Removes test environment created by Initialize-SimpleEnvironment.
 # Relies on environment variables (indirectly) - see
-# Remove-DockerNetworkAccordingToEnv, Stop-Agent, Disable-VRouterExtension.
+# Stop-Agent, Remove-DockerNetworkAccordingToEnv.
 function Remove-SimpleEnvironment {
     Write-Host "Removing simple test environment..."
-    Remove-DockerNetworkAccordingToEnv
     Stop-Agent
-    Disable-VRouterExtension
+    Remove-DockerNetworkAccordingToEnv

```

Comment:
The script does not start Agent because user may wish to run it under debugger.
When one wishes to clean the environment and vRouter Extension is no longer running, there's seemingly no useful scenario that involves running Agent, so it's reasonable to stop it.